src/main/java/org/jabref/gui/icon/JabRefMaterialDesignIcon.java

```diff
@@ -29,7 +29,8 @@ public enum JabRefMaterialDesignIcon implements Ikon {
     SET_CENTER("jab-setcenter", '\ue90b'),
     SET_ALL("jab-setall", '\ue90c'),
     VSCODE("jab-vsvode", '\ue90d'),
-    CANCEL("jab-cancel", '\ue90e');
+    CANCEL("jab-cancel", '\ue90e'),
+    SUBLIME_TEXT("jab-sublime-text", '\ue90f');
```

Comment:
Can we move this up to the other apps (WINEDT), so that the icons of the apps are grouped togehter? MAybe, sort the apps alphabetically?

Reply:
They are sorted by their unicode glyph